<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewWork</class>
 <widget class="QDialog" name="DialogNewWork">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Work</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Foucault2.qrc">
    <normaloff>:/Foucault2/Foucault2.ico</normaloff>:/Foucault2/Foucault2.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbWorkType">
       <item>
        <property name="text">
         <string>Undefined</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rough Grinding</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fine Grinding</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Polishing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Figuring</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QPlainTextEdit" name="pleWork"/>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dteWhen">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbHour"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbMinute"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbSecond"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Foucault2.qrc"/>
 </resources>
 <connections/>
</ui>
